server.go

```diff
@@ -24,5 +24,5 @@ func main() {
 
 	n := negroni.Classic()
 	n.UseHandler(router)
-	n.Run(":3000")
+	n.Run(":3001")
```

Comment:
Is this going to stay at 3001? Currently the front end is using 3001 but I can change it to whatever as well. Just wanted to know beforehand. Maybe we should have the front end use port 4000 so that we can increment easily.